knowledge-base/kb-scheduleview-howto-make-readonly.md

```diff
@@ -0,0 +1,119 @@
+---
+title: How To Make ScheduleView ReadOnly
```

Comment:
The 'title', 'description' and 'page_title' should contain different wording in order to optimize the search engines. You can The first two (title and description) are good, but you can think how to improve the 'page_title'.